soc/riscv/riscv-privilege/common/nuclei/nuclei_csr.h

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2019 Nuclei Limited. All rights reserved.
+ * Copyright (c) 2021 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
this should not be required when adding SPDX tag

Reply:
Removing it.

Reply:
Aha! I understood!
I was deleteing SPDX tag side...
Remains only SPDX tag and remove license description.